src/basics.ml

```diff
@@ -190,11 +199,12 @@ let iter_meta : bool -> (meta -> unit) -> term -> unit = fun b f ->
     | Vari(_)
     | Type
     | Kind
-    | Symb(_)    -> ()
+    | Symb(_)     -> ()
     | Prod(a,b)
-    | Abst(a,b)  -> iter a; iter (Bindlib.subst b Kind)
-    | Appl(t,u)  -> iter t; iter u
-    | Meta(v,ts) -> f v; Array.iter iter ts; if b then iter !(v.meta_type)
+    | Abst(a,b)   -> iter a; iter (Bindlib.subst b Kind)
+    | Appl(t,u)   -> iter t; iter u
+    | Meta(v,ts)  -> f v; Array.iter iter ts; if b then iter !(v.meta_type)
+    | LLet(t,a,u) -> iter a; iter (Bindlib.subst u t)
```

Comment:
This is not the intended behaviour, what if the bound variable does not occur?
Moreover, if it is used several times you are traversing `t` several times.
Use `iter t; iter a; iter (Bindlib.subst u Kind)`